instrumentation/camel-2.20/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/apachecamel/DirectCamelTest.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.apachecamel;
+
+import static io.opentelemetry.api.common.AttributeKey.stringKey;
+
+import io.opentelemetry.api.trace.SpanKind;
+import io.opentelemetry.instrumentation.testing.junit.AgentInstrumentationExtension;
+import io.opentelemetry.instrumentation.testing.junit.InstrumentationExtension;
+import org.apache.camel.CamelContext;
+import org.apache.camel.ProducerTemplate;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.springframework.boot.SpringApplication;
+import org.springframework.context.ConfigurableApplicationContext;
+
+class DirectCamelTest {
+
+  @RegisterExtension
+  public static final InstrumentationExtension testing = AgentInstrumentationExtension.create();
+
+  private static ConfigurableApplicationContext server;
+
+  @BeforeAll
+  static void setUp() {
+    SpringApplication app = new SpringApplication(DirectConfig.class);
+    server = app.run();
+  }
+
+  @AfterAll
+  static void cleanUp() {
+    if (server != null) {
+      server.close();
+      server = null;
+    }
+  }
+
+  @Test
+  void simpleDirectToSingleService() {
+
+    CamelContext camelContext = server.getBean(CamelContext.class);
+    ProducerTemplate template = camelContext.createProducerTemplate();
+
+    template.sendBody("direct:input", "Example request");
+
+    testing.waitAndAssertTraces(
+        trace ->
+            trace.hasSpansSatisfyingExactly(
+                span ->
+                    span.hasName("input")
+                        .hasKind(SpanKind.INTERNAL)
+                        .hasNoParent()
+                        .hasAttribute(stringKey("camel.uri"), "direct://input"),
```

Comment:
It would be better to use `hasAttributesSatisfyingExactly` instead of `hasAttribute`, groovy tests also verify that all attributes are asserted.

Reply:
thank you for the recommendation, I have updated accordingly